index.js

```diff
@@ -6,15 +6,9 @@ var dust = require('dustjs-linkedin');
 module.exports = function (opts) {
 	opts = opts || {};
 
-	if (opts.preserveWhitespace) {
-		dust.optimizers.format = function (ctx, node) {
-			return node;
-		};
-	}
-
-	if (opts.amd) {
-		dust.config.amd = true;
-	}
+	dust.config.whitespace = (opts.preserveWhitespace === true);
```

Comment:
Haven't looked at the Dust docs, but can't we just do `dust.config = opts;` now?